policy_for_val.tex

```diff
@@ -18,15 +18,15 @@ \subsection{Sammanfattning}
 Denna policyn beskriver hur valprocessen skall gå till både för val gjorda av sektionsmötet och styrelsen. Den täcker även hur processen inför valen skall gå till.
 
 \subsection{Syfte}
-Syftet med denna policy är att förtydliga stadgan och reglementet samt att vägleda valberedningen samt andra tillfälliga valberedningar i sitt arbete
+Syftet med denna policy är att förtydliga stadgan och reglementet samt att vägleda Valberedningen samt utskottsvalberedningarna i sitt arbete.
 
 \subsection{Omfattning}
 Alla val som behandlas på sektionen.
 
 \subsection{Historik}
 Utkast färdigställt av: Anna Qvil \\
 Ursprungligen antagen enligt beslut: VTM extra 2019.
-Uppdaterad: HTM1 2021, S13 2021, HTM2 2021 (enl. Policy för Policyer), VTM 2023, VTM-extra 2023. Uppdaterad enl. Policy för styrdokument på VTM-extra 2023. HTM1 2023.
+Uppdaterad: HTM1 2021, S13 2021, HTM2 2021 (enl. Policy för Policyer), VTM 2023, VTM-extra 2023. Uppdaterad enl. Policy för styrdokument på VTM-extra 2023. HTM1 2023. HTM2 2023.
```

Comment:
```suggestion
Uppdaterad: 
\begin{itemize}
\item HTM1 2021, 
\item S13 2021, 
\item HTM2 2021 (enl. Policy för Policyer), 
\item VTM 2023, VTM-extra 2023,
\item VTM-extra 2023 (Uppdaterad enl. Policy för styrdokument),
\item HTM1 2023,
\item HTM2 2023.
\end{itemize}
```

Reply:
Gör jag en ny commit för att den här justeringen ska gå igenom i PR? Är inte med på vad ovan betyder (tekniskt, förstår ändringen).

Reply:
Det som skulle vara snyggast skulle vara att commita dessa och sedan rebasea och squasha in dem i rätt commit.